scripts/pre_commit_linter.py

```diff
@@ -2477,6 +2538,27 @@ def _check_constants_declaration(self):
                             constants_to_source_filepaths_dict[
                                 constant_name] = filepath
 
+                # Checks that the *.constants.ts and the corresponding
+                # *.constants.ajs.ts file are in sync.
+                if filepath.endswith('.constants.ts'):
+                    angular_constants_nodes = (
+                        parsed_nodes[1].declarations[0].init.callee.body.body)
+                    for angular_constant_node in angular_constants_nodes:
+                        if not angular_constant_node.expression:
+                            continue
+                        angular_constant_name = (
+                            angular_constant_node.expression.left.property.name)
+                        angular_constants_set.add(angular_constant_name)
+
+                    angular_minus_angularjs_constants = (
+                        angular_constants_set.difference(
+                            angularjs_source_filepaths_to_constants_dict[
+                                corresponding_angularjs_filepath]))
+                    for constant in angular_minus_angularjs_constants:
+                        failed = True
+                        print ('%s --> The constant %s is not declared '
```

Comment:
We need to check the other way as well, right? I.e. angularjs - angular?

Reply:
Actually, this will be automatically checked by the condition [here](https://github.com/oppia/oppia/pull/7066/files#diff-3093ff31fd2fc587c0d99fca7d3afdf0R2485).